packages/shared/src/components/MainFeedLayout.tsx

```diff
@@ -272,6 +272,25 @@ export default function MainFeedLayout({
       return null;
     }
 
+    if (user?.defaultFeedId && user.defaultFeedId !== user.id) {
+      return {
+        feedName: SharedFeedPage.MyFeed,
+        feedQueryKey: generateQueryKey(SharedFeedPage.MyFeed, user),
+        query: CUSTOM_FEED_QUERY,
+        variables: {
+          feedId: user.defaultFeedId,
+          feedName: SharedFeedPage.MyFeed,
+        },
+        emptyScreen: propsByFeed[feedName].emptyScreen || <FeedEmptyScreen />,
+        actionButtons: feedWithActions && (
```

Comment:
I think some padding is missing (vs what we have live)
<img width="1042" alt="image" src="https://github.com/user-attachments/assets/519dc645-e953-4f31-94f6-0ad6862774ad" />

Reply:
@capJavert Its unrelated to this PR, as it is the same on [AS-814-custom-feeds-plus](https://github.com/dailydotdev/apps/tree/AS-814-custom-feeds-plus), but I can tell that somewhere along the line here the navbar has increased from 57px height to 65px height, which is why it looks different. I tried some quick things, but didn't seem to work. Maybe we should make a bug task on it ?

Reply:
if thats the case, here is the task https://dailydotdev.atlassian.net/browse/AS-865